File: client/src/app/shared/models/end-of-day.ts
export interface EndOfDay {
  adjClose: number
  adjHigh: number
  adjLow: number
  adjOpen: number
  adjVolume: number
  close: number
  date: string
  divCash: number
  high: number
  low: number
  open: number
  splitFactor: number
  volume: number
}

export interface EndOfDayMeta {
  ticker: string
  name: string
  exchangeCode: string
  startDate: string
  endDate: string
  description: string
}

export class EndOfDayParams {
  startDate?: string;
  endDate?: string;
  //'daily', 'weekly','monthly', 'annually'
  resampleFreq = 'annually';
}

File: API/DTOs/EndOfDay.cs
namespace API.DTOs
{
    public class EndOfDay
    {
        public double AdjClose { get; set; }
        public double AdjHigh { get; set; }
        public double AdjLow { get; set; }
        public double AdjOpen { get; set; }
        public long AdjVolume { get; set; }
        public double Close { get; set; }
        public DateTime Date { get; set; }
        public double DivCash { get; set; }
        public double High { get; set; }
        public double Low { get; set; }
        public double Open { get; set; }
        public double SplitFactor { get; set; }
        public long Volume { get; set; }
    }
}